storage_file/__manifest__.py

```diff
@@ -9,6 +9,7 @@
     "category": "Storage",
     "website": "https://github.com/OCA/storage",
     "author": " Akretion, Odoo Community Association (OCA)",
+    "maintainers": ["sebastienbeau", "hparfr", "simahawk"],
```

Comment:
@sebastienbeau @hparfr you don't mind right? :smile:

Reply:
Ok for us ;)